services/site/src/pages/about.tsx

```diff
@@ -0,0 +1,193 @@
+              </p>
+            </div>
+          </div>
+
+          <p className="my-8">
+            In an ideal world, all websites are designed and developed so everyone can use them. Sadly, many websites do not comply with web
+            accessibility guidelines. Therefore, especially people with impairments and disabilities are barred from using them. One of the main
+            problems is that many developers don't know enough about web accessibility. <span aria-label="Allyphant">a11yphant</span> wants to improve
+            this situation by giving them a tool for learning accessibility in a fun, interactive way.
+          </p>
+
+          {prefersReducedMotion ? (
+            <Image
+              alt="The a11yphant website where users can complete interactive challenges on web accessibility."
+              src={a11yphantStatic}
+              quality={100}
+              sizes="100vw"
+              style={{
+                objectFit: "cover",
+              }}
+            />
+          ) : (
+            <Image
+              alt="The a11yphant website where users can complete interactive challenges on web accessibility."
+              src={a11yphantAnimation}
+              quality={100}
+              sizes="100vw"
+              style={{
+                objectFit: "cover",
+              }}
+            />
+          )}
+
+          <h2 className="h4 mt-8">
+            But what is{" "}
+            <span aria-label="Allyphant" className="text-2xl">
+              a11yphant
+            </span>
+            ?
+          </h2>
+
+          <p>
+            <span aria-label="Allyphant">a11yphant</span> is an interactive online course for web accessibility. In this course, you will revisit web
+            development topics from an accessibility perspective. For example, how to make sure that assistive technologies like screen readers can
+            interact with the website? Or, what is the purpose of the different semantic HTML elements? These are some of the things you will learn by
+            writing meaningful markup and completing quizzes.
+          </p>
+
+          <h2 className="h4 mt-8">Coding-Challenges and Quizzes</h2>
+
+          <p>
+            Various accessibility topics are split up into short challenges. On <span aria-label="Allyphant">a11yphant</span>, you solve coding levels
+            and single-choice quizzes in each challenge. Each level teaches you one thing at a time. Our evaluation provides you with detailed
+            feedback on our success criteria. If you struggle to solve a challenge, hints will help you out.
+          </p>
+
+          <h2 className="h4 mt-8">Meet the team</h2>
+
+          <p>
+            <span aria-label="Allyphant">a11yphant</span> started as a master’s project created by six students at the{" "}
+            <a href="https://www.fh-salzburg.ac.at/en/" target="_blank" rel="noopener noreferrer nofollow">
```

Comment:
All in-text links need the following class:

className={clsx(
  "text-light font-sans font-normal border-light",
  "transition-colors duration-300",
  "hover:text-primary-light hover:border-transparent",
  "focus-rounded-instead-of-underline",
)}